packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -119,7 +118,7 @@ export default class RelationControl extends React.Component {
       const metadata = {};
       const searchFieldsArray = getSearchFieldArray(field.get('search_fields'));
       const { payload } = await query(forID, collection, searchFieldsArray, '', file);
-      const hits = payload.response?.hits || [];
```

Comment:
This has the potential to break existing widgets that use the `query` API.
Since it's not an officially documented one I think this is ok.